docs/en/sql-reference/sql-statements/Data Definition/CREATE TABLE.md

```diff
@@ -296,9 +296,11 @@ Syntax:
         ```
         PROPERTIES (
             "storage_medium" = "[SSD|HDD]",
+            ["storage_cold_medium" = "[HDD|S3]"],
+            ["remote_storage_resource" = "xxx"],
             ["storage_cooldown_time" = "yyyy-MM-dd HH:mm:ss"],
             ["replication_num" = "3"],
-			["replication_allocation" = "xxx"]
+            ["replication_allocation" = "xxx"]
```

Comment:
what does replication_allocation means?

Reply:
if it is the same with replication_num why add this param?

Reply:
You can refer to #5902

Reply:
I see.